pom.xml

```diff
@@ -181,7 +181,6 @@
         <module>grpc-proto</module>
         <module>grpc</module>
         <module>metrics-jersey</module>
-        <module>grpc-examples</module>
```

Comment:
Removing this from the parent pom since I'm not sure why we need to build examples along with the rest of the project.

Reply:
If there is a way to keep it in some setting (profile), one benefit there'd be would be to ensure examples still compile?
I agree that packaging and releasing does not make much sense, not sure how to achieve just compilation (and if any tests exist, running them).